githublib.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+import requests
+import json
+import time
+from datetime import datetime
+
+from common import get_envvar
+
+SECONDS_PER_WEEK = 604800
+
+
+class Github:
+    def __init__(self, token, owner, repo, expected_label):
+        self.owner = owner
+        self.repo = repo
+        self.expected_label = expected_label
+        self.query_url = f"https://api.github.com/repos/{owner}/{repo}/issues"
+        self.headers = {"Authorization": f"token {token}"}
+
+    def get_issues(self, page):
+        params = {"state": "open", "page": page, "per_page": "100"}
+
+        r = requests.get(self.query_url, headers=self.headers, params=params)
+        data = r.json()
+        return data
+
+    def check_labels(self, issue):
```

Comment:
I do not understand what this does really. You want to filter by label?

Reply:
we filter for example only issues that are `sig/network`
we don't process others

Reply:
Ok, so this is just an inner helper (and not public). The operation is "filter".